app/src/main/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionModel.kt

```diff
@@ -14,17 +14,24 @@ data class TeleconsultSharePrescriptionModel(
     val medicines: List<PrescribedDrug>?,
     val medicalRegistrationId: String?,
     val patientProfile: PatientProfile?,
-    val downloadButtonState: ButtonState?
+    val downloadButtonState: ButtonState?,
+    val shareButtonState: ShareButtonState?
 ) : Parcelable {
 
   companion object {
-    fun create(patientUuid: UUID, prescriptionDate: LocalDate, downloadButtonState: ButtonState?) = TeleconsultSharePrescriptionModel(
+    fun create(
+        patientUuid: UUID,
+        prescriptionDate: LocalDate,
+        downloadButtonState: ButtonState?,
+        shareButtonState: ShareButtonState?
```

Comment:
We don't have to pass the share button state to the factory method.

Reply:
We're adding the default state as NOT_SHARING and not null.

Reply:
same comment as download button state